IndexedDB/idbfactory_open2.htm

```diff
@@ -6,12 +6,12 @@
 <script src=support.js></script>
 
 <script>
-    var open_rq = createdb(async_test(), 'database_name', 13);
+    var open_rq = createdb(async_test(), 'database_name_open2', 13);
```

Comment:
How about e.g. `document.location + '-database_name'` to avoid trying to keep the suffix unique across tests and defend against future copy/paste?

Reply:
[The utility has functionality to avoid name reuse](https://github.com/web-platform-tests/wpt/blob/198d075203a6734aa5469f8c7e1fa7d6c1e234bc/IndexedDB/support.js#L40), but I personally prefer a more explicit value and (maybe idealistically) hope future reviewers will scrutinize new tests.

With that said, I'm happy to change this as per your preference. Would you like to use `document.location` or fall back to the utility-provided mechanism?

Reply:
FWIW, the somewhat more recently written `indexeddb_test` helper uses location plus the test name which should be guaranteed unique (and also stable across runs)

I'd prefer to avoid the helper you link to (random + timestamp); but I really don't trust that reviewers will see a string and ensure it is unique across files or correlates with the filename.

So... either something like what I suggest, or factor out a new helper? Not picky about which, though.

Reply:
You got it!